src/ansible_navigator/ui_framework/field_curses_information.py

```diff
@@ -0,0 +1,54 @@
+"""An information field formatted as curses lines."""
+from dataclasses import dataclass
+from typing import Callable
+from typing import Optional
+from typing import Union
+
+from .curses_defs import CursesLines
+from .curses_window import Window
+from .form_handler_information import FormHandlerInformation
+from .sentinels import Unknown
+from .sentinels import unknown
+from .validators import FieldValidators
+
+
+@dataclass
+class FieldCursesInformation:
+    """An information field mad of curses lines."""
```

Comment:
Should it be `made` instead of `mad` :-)

Reply:
:)